frontend/src/app/shared/lead.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@angular/core';
+import {HttpClient, HttpHeaders } from '@angular/common/http';
+import { Lead } from '../leads/model/lead.model';
+import { Observable, of } from 'rxjs';
+import { catchError, map, tap } from 'rxjs/operators';
+
+
+@Injectable()
+export class LeadService {
+private url = 'http://localhost:8080/api/webhook';
```

Comment:
No need to prepend base url.

Reply:
That causing problem for me let me see if there is no any error with just  /api/webhook